{{ cookiecutter.repo_name }}/.devcontainer/README.md

```diff
@@ -1,10 +1,8 @@
 # FFRD Devcontainer Template
 Devcontainers allow interactive development inside of a docker container using VSCode. 
 
-
 This devcontainer creates a reproducible environment for python projects using micromamba    
-environments (faster/more robust version of conda). To add this devcontainer template to your project, copy this .devcontainer folder  
-into the parent directory of your repository, and copy this [.gitattributes file] into the same parent directory.
+environments (faster/more robust version of conda). 
 
 When opening this repository in VSCode, you may be prompted to re-open the project in devcontainer.  
```

Comment:
Active tone is typically a best practice for software/business documentation.  -->  ie: _"You may be prompted to re-open the project in a devcontainer when opening this repositiory in VSCode."_

- [ ] Use active tone, instead of passive tone